fenix/app/src/main/res/values-night/colors.xml

```diff
@@ -151,4 +151,7 @@
     <!-- Tab Counter colors -->
     <color name="mozac_ui_tabcounter_default_tint" tools:ignore="UnusedResources">@color/fx_mobile_icon_color_primary</color>
     <color name="mozac_ui_tabcounter_default_text" tools:ignore="UnusedResources">@color/fx_mobile_text_color_primary</color>
+
+    <!-- Cookie Banner Handling -->
+    <color name="cookie_banner_handling_details_report_a_site_text_color">#CB9EFF</color>
```

Comment:
We don't add new colours for specific features of the UI, designers  reuse the colour palette that we already defined. I think for this case we need to use `Accent` in Figma they include the name of the colour that we should use. @gabrielluong could add more details :).

![image](https://user-images.githubusercontent.com/773158/228679889-2d283436-115c-47ba-8b16-b032502033c7.png)

Reply:
Right, we have a design system with color variables in Fenix. So, there should be no need to add colors manually. We should be using `@color/fx_mobile_text_color_accent`

Reply:
Done